src/devices/Ina219/Ina219Enum.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Adc
+{
+    public partial class Ina219
+    {
+        /// <summary>
+        /// An enumeration representting the registers exposed by the INA219 device via I2c.
+        /// </summary>
+        protected enum Register : byte
```

Comment:
1. Anything which is not in the public APIs should be private - if it needs to be shared with other sensors make it internal if possible, otherwise make it public
2. Do not nest types under the classes